dev-docs/bidders/metax.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: bidder
+title: MetaX
+description: MetaX Prebid Bidder Adapter
+biddercode: metax
+tcfeu_supported: true
+gvl_id: 1301
+usp_supported: true
+coppa_supported: true
+gpp_supported: true
```

Comment:
Please use `gpp_sids`

> If you support the IAB’s GPP consent string, add gpp_sids with a comma separated list of section names, e.g. gpp_sids: tcfeu, tcfca, usnat, usstate_all, usp. Default is None.

from https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

Reply:
thanks for reminding. move to gpp_sids now